pkgs/by-name/pa/parmetis/package.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, stdenv
-, fetchurl
-, cmake
-, mpi
+{
+  lib,
+  stdenv,
+  fetchurl,
+  cmake,
+  mpi,
 }:
 
 stdenv.mkDerivation rec {
   pname = "parmetis";
   version = "4.0.3";
 
   src = fetchurl {
-    url = "http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/parmetis-${version}.tar.gz";
-    sha256 = "0pvfpvb36djvqlcc3lq7si0c5xpb2cqndjg8wvzg35ygnwqs5ngj";
+    url = "https://web.archive.org/web/20221116225811/http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/parmetis-${version}.tar.gz";
+    hash = "sha256-8tmiMbfPl/H+5ujJZjET6/bCQNQH08EYxVs2M9a+bl8=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ mpi ];
 
+  enableParallelBuilding = true;
+
```

Comment:
```suggestion
```

the cmake setup hook already sets this

Reply:
It would be nice to include `ninja` in `nativeBuildInputs` for faster builds, though, if it works.

Reply:
unsure, when building without this commit:

```
$ time nix-build -A parmetis
nix-build -A parmetis  1.04s user 0.38s system 3% cpu 46.465 total
```


when building with this commit:
```
$ time nix-build -A parmetis
nix-build -A parmetis  1.09s user 0.36s system 13% cpu 11.118 total
```

Reply:
I think it might just be Ninja that defaults to parallel builds? I would suggest adding `ninja` next to `cmake` in `nativeBuildInputs`, which should be faster than even a parallel Make build.

Reply:
```
error: builder for '/nix/store/br8kfshh9y3jgkx7gw5f2rkx3392nzwr-parmetis-4.0.3.drv' failed with exit code 1;
       last 10 log lines:
       > -- Looking for execinfo.h - found
       > -- Looking for getline
       > -- Looking for getline - found
       > -- checking for thread-local storage - found
       > -- Configuring done (3.2s)
       > -- Generating done (0.0s)
       > -- Build files have been written to: /build/parmetis-4.0.3/build/Linux-x86_64
       > Running phase: buildPhase
       > build flags: -j12
       > ninja: error: loading 'build.ninja': No such file or directory
       For full logs, run 'nix log /nix/store/br8kfshh9y3jgkx7gw5f2rkx3392nzwr-parmetis-4.0.3.drv'.
nix-build -A parmetis  1.05s user 0.42s system 27% cpu 5.408 total
```